docs/install/assets/eventing.md

```diff
@@ -0,0 +1,11 @@
+# Knative Serving installation files (0.21 releases)
+
+| File name | Description | Dependencies|
+| --- | --- | --- |
+| eventing-core.yaml | Knative Serving core components. |  none |
+| eventing-crds.yaml | Knative Serving core CRDs. | none |
+| eventing-post-install.yaml | Optional jobs after installing eventing-core.yaml. | eventing-core.yaml |
```

Comment:
I would try to get a better description, on why would someone needs to use this job, is it to be use as part of the upgrade instructions? Only applicable when doing upgrades?

Reply:
Yep, I wasn't sure exactly what the jobs do, so we used the same description from the PR for serving https://github.com/knative/docs/pull/3296/files
Having said that, it would definitely be useful to get a clearer definition

@csantanapr  who, would be the best person to chime in on this ?